WordPress/Classes/Utility/ImmuTable.swift

```diff
@@ -279,6 +279,14 @@ public class ImmuTableViewHandler: NSObject, UITableViewDataSource, UITableViewD
         return cell
     }
 
+    public func tableView(tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return viewModel.sections[section].headerText
+    }
+
+    public func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        return viewModel.sections[section].footerText
```

Comment:
One alternative is to have a delegate and hook into `willDisplayHeaderView`/`willDisplayFooterView` for customisation. I've tried this approach for My Profile here https://gist.github.com/koke/cad5c7ebcf9988b6909e

It works for setting the font/color, but there's no way to set the insets without a custom view.If we need to keep the custom insets, another option would be to pass the `viewForHeaderInSection:` t to the delegate, but then the delegate would be responsible for setting the right title.